Tools/clinic/clinic.py

```diff
@@ -3000,7 +3014,7 @@ def pre_render(self):
         """
         pass
 
-    def parse_arg(self, argname, displayname):
+    def parse_arg(self, argname: str, displayname: str):
```

Comment:
```suggestion
    def parse_arg(self, argname: str, displayname: str) -> str:
```

Reply:
IIRC, this caused a lot of havoc, so I'd rather save it for later :)

Reply:
Oh, sure!